ark/phenotyping/som_utils_test.py

```diff
@@ -0,0 +1,101 @@
+import os
+import pytest
+import tempfile
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import ark.phenotyping.som_utils as som_utils
+import ark.utils.misc_utils as misc_utils
+import ark.utils.test_utils as test_utils
+
+
+def mocked_cluster_pixels(base_dir, chan_list):
```

Comment:
The way you have this set up, the mocked function won't report errors if we created a totally garbage csv with none of the right columns and no values. 

The mocked function should as closely as possible replicate the original function, so that we catch as many errors as possible. 

We should instead actually read in the csv file, subset based on the provided column names, do a simple manipulation with all of the values in the remaining columns, use that manipulation to create cluster labels, and then save the result

Reply:
What kind of manipulation are we talking about? The SOM process doesn't modify any of the channel values. I'm currently using the average of each row (because the test matrix values are uniform over [0, 1)) to generate dummy cluster IDs.

Reply:
Yeah, I think average makes sense.